packages/desktop-client/src/components/schedules/EditSchedule.js

```diff
@@ -487,7 +487,7 @@ export default function ScheduleDetails({ modalProps, actions, id }) {
               }}
               style={{
                 padding: '0 10px',
-                color: colors.n5,
+                color: theme.pageText,
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏wrong color

Before:
<img width="256" alt="Screenshot 2023-08-12 at 09 28 07" src="https://github.com/actualbudget/actual/assets/886567/143ac76b-c47f-47a7-b5dc-d52a23baef56">


After:
<img width="220" alt="Screenshot 2023-08-12 at 09 28 12" src="https://github.com/actualbudget/actual/assets/886567/381648e5-34d3-44aa-a8e8-1daed740296d">